examples/channels-example/service/Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV NODE_ENV=production
 COPY package.json /opt/service/package.json
 
 # Install all dependencies
-RUN npm i
+RUN npm i --no-package-lock
```

Comment:
why?

Reply:
There were some warnings in logs + it doesn't really make sense to generate lock file in this case imo.